src/cogs/jeopardy.py

```diff
@@ -0,0 +1,83 @@
+import asyncio
+import random
+from fuzzywuzzy import fuzz
+import discord
+from discord.ext import commands
+import json
+
+from src.bot import config
+
+
+class Jeopardy(commands.Cog):
+    """A Jeopardy question answer game utilizing Levenshtein Distance to calculate answer matching (beta)."""
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+        self.attempts = 0
+
+    @commands.max_concurrency(1, per=commands.BucketType.guild, wait=False)
+    @commands.command()
+    async def jeopardy(self, ctx):
+        """Generates a Jeopardy question from over 200,000 historical Jeopardy Questions."""
+        try:
+            with open('JEOPARDY_QUESTIONS1.json') as f:
+                data = json.loads(f.read())
+                position = random.randint(0, 199999)
+                question = data[position]['question']
+                answer = data[position]['answer']
```

Comment:
It's already pretty readable without all these lines, maybe merge them into one or two